Add OLS-4 Grad dates and OLS-5 announcement (#335)
open-life-science/open-life-science.github.io: _posts/2021-12-06-announcing-applications-for-ols-5.md

These calls will be hosted for the OLS members via Zoom and livestreamed for everyone on YouTube with real-time captions using Otter.ai.
- Slot 1: 18 January, Tuesday - 09:00 - 10:00 AM UTC
    - ⏰ [In your timezone]((https://arewemeetingyet.com/London/2022-01-18/09:00/week-16-ols-4-grad1))
- Slot 2: 19 January, Wednesday - 14:00 - 15:00 UTC
    - ⏰ [In your timezone]((https://arewemeetingyet.com/London/2022-01-19/14:00/week-16-ols-4-grad1))
- Slot 3: 20 January, Thursday - 17:00 - 18:00 UTC
    - ⏰ [In your timezone]((https://arewemeetingyet.com/London/2022-01-20/17:00/week-16-ols-4-grad1))

We especially welcome you if you are:
- friends, family, colleagues, of the current project leaders who are graduating.
- planning to apply for OLS-5 and want to see what our current projects are like.
- interested in learning about the OLS and what we do.
- from an open research institute, funder, journal, or anyone else who would like to see some of the shining stars of early open science projects from this cohort.

## Call for application for OLS-5 is NOW OPEN!

Interested in applying for the next round? We can't wait to see your application! 

> Please submit your application via [OpenReview](https://openreview.net/group?id=openlifesci.org/Open_Life_Science/2022/Cohort_5) before the end of the day on **15 January 2022** (hard deadline).

Reviewer: Let's just make sure not to surprise people w/ open review deadline for registration. 

```suggestion
Please submit your application via [OpenReview](https://openreview.net/group?id=openlifesci.org/Open_Life_Science/2022/Cohort_5) before the end of the day on **15 January 2022** (hard deadline). **Important note:** The deadline for registering for OpenReview is 12 January, as all OpenReview accounts are approved manually. 
```

To learn about the timeline, process and details for the OLS-5, visit our homepage: [openlifesci.org/](https://openlifesci.org/) and 
[GitHub repo with application guidelines](https://github.com/open-life-science/application-forms).

> We encourage you to watch the OLS graduations from last three cohorts [on YouTube](https://www.youtube.com/openlifesci) to get a sense of what the project will be like (direct link to [OLS-3 graduation calls](https://www.youtube.com/playlist?list=PL1CvC6Ez54KD01eg-XVq0AUHNEpG9dnrA)).

Reviewer: ```suggestion
We encourage you to watch the OLS graduations from the last three cohorts [on YouTube](https://www.youtube.com/openlifesci) to get a sense of what the project will be like (direct link to [OLS-3 graduation calls](https://www.youtube.com/playlist?list=PL1CvC6Ez54KD01eg-XVq0AUHNEpG9dnrA)).
```

> For OLS-5 we'll be offering micro-grants as suggested for the [OLS-3 and OLS-4]({% link _posts/2020-11-23-micro-grants-in-2021-ols-3-ols-4.md %}) to help grow participation and inclusion, and to support budding new projects. More details regarding the budget will be shared in a separate blog post.

Reviewer: ```suggestion
For OLS-5 we'll be offering micro-grants, as we did in [OLS-3 and OLS-4]({% link _posts/2020-11-23-micro-grants-in-2021-ols-3-ols-4.md %}) to help grow participation and inclusion, and to support budding new projects. More details regarding the budget will be shared in a separate blog post.
```

Our amazing project leads for OLS-4 brought over 20 projects from around the world and have been working hard amid the pandemic to advance their knowledge in open and reproducible research practices while establishing themselves 
as experts in their fields ([see our projects and participants]({% link _ols-4/projects-participants.md %})). 
We are lucky enough to witness their projects and network grow while working alongside our mentors, expert speakers, 
and call facilitators.

It's time to set the stage for them to share their projects with the wider open science communities through our graduation calls on 18, 19 and 20 January 2022.
We invite you to come and watch the [live-streamed calls on YouTube]({{site.youtube}}) - they'll be spread across three different 
hour-long sessions, and available on YouTube after the event if you can't attend in real-time. 

These calls will be hosted for the OLS members via Zoom and livestreamed for everyone on YouTube with real-time captions using Otter.ai.

**Please register for one (or all) graduation calls via the [Eventbrite pages](https://www.eventbrite.com/cc/ols-4-graduations-44929)**.
- Slot 1: 18 January, Tuesday - 09:00 - 10:00 AM UTC

>     - ⏰ [In your timezone]((https://arewemeetingyet.com/London/2022-01-18/09:00/week-16-ols-4-grad1))
> - Slot 2: 19 January, Wednesday - 14:00 - 15:00 UTC
>     - ⏰ [In your timezone]((https://arewemeetingyet.com/London/2022-01-19/14:00/week-16-ols-4-grad1))
> - Slot 3: 20 January, Thursday - 17:00 - 18:00 UTC
>     - ⏰ [In your timezone]((https://arewemeetingyet.com/London/2022-01-20/17:00/week-16-ols-4-grad1))

Reviewer: ```suggestion
    - ⏰ [In your timezone](https://arewemeetingyet.com/London/2022-01-18/09:00/week-16-ols-4-grad1)
- Slot 2: 19 January, Wednesday - 14:00 - 15:00 UTC
    - ⏰ [In your timezone](https://arewemeetingyet.com/London/2022-01-19/14:00/week-16-ols-4-grad1)
- Slot 3: 20 January, Thursday - 17:00 - 18:00 UTC
    - ⏰ [In your timezone](https://arewemeetingyet.com/London/2022-01-20/17:00/week-16-ols-4-grad1)
```
